### DISCUSSION TITLE
Modbus TCP Port  1502   or other  502

### DISCUSSION THREAD

**Discussion by hno2fe** [WillCodeForCats/solaredge-modbus-multi#504] [General]
Is it important that the Modbus TCP port is on 1502?
Port 502 was preconfigured on my system!

SE10K-RWB48BFN4 with Bui

![grafik](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/145226308/65435de1-74b3-47bd-b9fe-5945c2f154e1)

**Comment by WillCodeForCats** [comment]
You can use any port.

**Reply by hno2fe** [reply]
Thanks for the Information
